streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala

```diff
@@ -145,6 +146,10 @@ private[ui] class StreamingPage(parent: StreamingTab)
   private val listener = parent.listener
   private val startTime = System.currentTimeMillis()
 
+  parent.parent.attachHandler(
+    StreamingApiRootResource.getServletHandler(parent.parent, listener, startTime)
```

Comment:
maybe i can add a comment to explain the current code for now
and if necessary, open an issue to discuss where to record the startTime